server/server.go

```diff
@@ -313,6 +325,15 @@ func (server *HTTPServer) Initialize(address string) http.Handler {
 
 	apiPrefix := server.Config.APIPrefix
 
+	if server.Config.EnableCORS {
+		router.Use(server.addDefaultHeaders)
```

Comment:
maybe it's better to call it addCORSHeaders?

Reply:
Yep, it sound better